katago_server/startposes/models/startpos.py

```diff
@@ -48,9 +48,10 @@ class StartPos(Model):
     objects = StartPosQuerySet.as_manager()
 
     class Meta:
-        verbose_name = _("StartPosesNetwork")
-        verbose_name_plural = _("Networks")
+        verbose_name = _("StartPos")
```

Comment:
nits: I will name that `StartPosition` or `InitialPosition`
It was not clear for me atfirst what is was.

Reply:
I'll keep the name the same since it also is consistent with KataGo client parameters.